docs/UserGuide.adoc

```diff
@@ -80,12 +80,30 @@ can be used to add multiple ingredients and its respective quantities in the sam
 * Parameters can be in any order. +
 (e.g. if the command specifies `n/NAME t/TAG i/INGREDIENT q/QUANTITY`,
 `q/QUANTITY t/TAG i/INGREDIENT` is also acceptable).
+* Parameters cannot start with spaces.
+
+|===
+|Parameter type | Constraint
+
+|INDEX
+|Must be a whole number.
+
+|INGREDIENT
+|Can contain alphabets, numbers, and spaces, as well as the symbols `'`, `-`, and `%`.
+
+|QUANTITY
+|Must start with a numerical value, which can contain decimal points or slashes to represent a fraction. The value can
+be followed by an optional unit that only consists of alphabets, possibly separated by spaces.
+
+|TAG
+|Can only contain alphabets and numbers. No spaces are permitted.
+|===
```

Comment:
Good call on setting a table for the limitations of each parameter. I have added a small line with regards to the limitations on my other commit, might act as a good preface for your table.